explorer/testdata/operator/operator-modulo.carbon

```diff
@@ -0,0 +1,56 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{explorer} %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes=false %s
+// RUN: %{explorer} --parser_debug --trace_file=- %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes %s
+// AUTOUPDATE: %{explorer} %s
+// CHECK: result: 0
+
+package ExplorerTest api;
+
+fn identFn( x: i32 ) -> i32{
+    return x;
+}
+
+fn Main() -> i32 {
+  var x : i32 = 7 % 2;
+
+
+  if(  not (x == 1) ){
+      return 1;
+  }
+  x = 3*3 % 3;
+  if( not (x == 0)){
+      return 2;
+  }
+
+  x = 4+4+4+4 % 2+1;
+  if(  not (x == 1) ){
+      return 3;
+  }
+
+  x = 15 % identFn(2)+1;
```

Comment:
These three examples are all invalid per the currently-approved design and require parentheses; see [proposal 1083](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p1083.md#use-the-same-precedence-group-for-modulo-and-multiplication) for details.

Reply:
I must admit that I have not seen this proposal. I figured the C++ operator precedence would be fine